components/avatar/avatar.css

```diff
@@ -0,0 +1,80 @@
+:root{
+    --blue-violet:blueviolet;
+    --orange:orange;
+    --pink:pink;
+
+
+}
+.avatar {
+    vertical-align:middle;
+    font-size: 1.5rem;
+    color:white;
+    margin-bottom: 4rem;
+}
+.avatar-container{
+  margin-left:0;
+  border:1px solid grey;
+  padding:1.56rem;
+  width:80%;
```

Comment:
Instead of % can use rem here, % references width relative to the parent container which can be a problem when the application scales

Reply:
ok i will take care of it,thnks for review